homeassistant/components/prometheus.py

```diff
@@ -29,8 +30,13 @@
 DOMAIN = 'prometheus'
 DEPENDENCIES = ['http']
 
+CONF_PROM_NAMESPACE = 'namespace'
+PROM_NAMESPACE = 'hass'
+
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: recorder.FILTER_SCHEMA,
+    DOMAIN: recorder.FILTER_SCHEMA.extend({
```

Comment:
Use the origin filter schema from https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entityfilter.py